vfnext/bin/compileOutput.py

```diff
@@ -0,0 +1,510 @@
+#!/usr/bin/env python
+
+from ast import Assert
+import os
+import pandas as pd
+import sys
+from Bio import SeqIO
+import argparse
+
+__author__ = "Antonio Marinho"
+__license__ = "GPL"
+__version__ = "0.1"
```

Comment:
se formos adotar o SemVer, da pra adotar aqui também
```suggestion
__version__ = "0.1.0"
```